armi/settings/fwSettings/globalSettings.py

```diff
@@ -627,6 +628,14 @@ def defineSettings() -> List[setting.Setting]:
             "setting the powerFractions setting.",
             schema=vol.All(vol.Coerce(float), vol.Range(min=0)),
         ),
+        setting.Setting(
```

Comment:
That was my original idea. But I'm torn.  People tend to WRITE to the settings during a run.  My concern is that will happen and someone will try to use the settings they get from the DB to restart a run and it will throw an error.

I'm not 100% sure that will happen, of course. Hmm

I suppose it could still be a warning log statement.